docs/examples/bci2000/index.md

```diff
@@ -0,0 +1,24 @@
+# BCI2000
+
+BCI2000 is an open-source, all around purpose software designed for research with brain-computer interface (BCI).
```

Comment:
Grammar improvement:

`BCI2000 is an open-source, fully-featured software designed to support Brain-Computer Interface (BCI) research. It is developed and maintained by the National Center for Adaptive Neurotechnologies (NCAN). The software has been used for decades and offers a wide range of tools to support common BCI experiments.`